pay-api/src/pay_api/services/eft_short_names.py

```diff
@@ -75,16 +75,6 @@ class EFTShortnamesSearch:  # pylint: disable=too-many-instance-attributes
 class EFTShortnames:  # pylint: disable=too-many-instance-attributes
     """Service to manage EFT short name model operations."""
 
-    @staticmethod
-    def get_eft_credit_balance(short_name_id: int) -> Decimal:
```

Comment:
Moved to model due to cyclic issues with statement service.

As previously discussed we will be refactoring this service to split it out later. Depending on the query some of these functions are probably better in the model.